sched/handle_request.cpp

```diff
@@ -1416,13 +1416,12 @@ void process_request(char* code_sign_key) {
             if (config.min_sendwork_interval) {
                 double diff = dtime() - last_rpc_time;
                 if (diff < config.min_sendwork_interval) {
-                    double t_left = config.min_sendwork_interval - diff;
                     ok_to_send_work = false;
                     log_messages.printf(MSG_NORMAL,
-                        "Not sending work - last request too recent, please wait: %f\n", t_left
+                        "Not sending work - last request too recent, please wait: %f\n", config.min_sendwork_interval
```

Comment:
This means that user will always see the, let's say, 'default' delay interval even if there is less time left before he can do next request.
E.g.
There is a default delay of 60 sec.
User waits 59 seconds, make another request and see the message that he should wait for 60 seconds more while in fact he had to wait only 1 second before doing next request. Isn't this misleading a little bit?

Reply:
> he had to wait only 1 second

That's the problem - he didn't. The 1-second message came in an RPC, and the sending of that RPC re-started the timer. He **does** have to wait the full 60 seconds.

Reply:
Ok, I see. No questions then